.github/workflows/ansible-test.yml

```diff
@@ -0,0 +1,59 @@
+# README FIRST
+# 1. replace "NAMESPACE/COLLECTION_NAME" with the correct name, ie "community/zabbix"
+# 2. If you don't have unit tests remove that section
+# 3. If your collection depends on other collections ensure they are installed, see "Install collection dependencies"
+# If you need help please ask in #ansible-devel on Freenode IRC
```

Comment:
Who's readme is this? Seems like this needs to be reworded.

Reply:
Removed this. This was leftover comment cruft from Ansible's Github action example.